plantseg/gui/gui_widgets.py

```diff
@@ -143,7 +144,7 @@ def __init__(self, frame, config, col=0, module_name="preprocessing", font=None,
                                                          _type=str,
                                                          _font=font),
                            "factor": RescaleEntry(self.preprocessing_frame,
-                                                  text="Rescaling: ",
+                                                  text="Rescaling (x,y,z):",
```

Comment:
should be `(z,y,x)` right?

Reply:
I think we always used (z, x, y) I'll keep in this way for now..